src/AutoMerge/new-package.jl

```diff
@@ -84,47 +13,10 @@ function pull_request_build(data::GitHubAutoMergeData, ::NewPackage; check_licen
     this_pr_can_use_special_jll_exceptions =
         this_is_jll_package && data.authorization == :jll
 
-    # Each element is a tuple of a guideline and whether it's
-    # applicable. Instead of a guideline there can be the symbol
-    # `:update_status` in which case the PR status will be updated
-    # with the results so far before continuing to the following
-    # guidelines.
-    guidelines =
-        [
-            (guideline_registry_consistency_tests_pass, true),  # 1
-            (guideline_pr_only_changes_allowed_files, true),    # 2
-            # (guideline_only_changes_specified_package, true), # 3 (unimplemented)
-            (guideline_normal_capitalization,                   # 4
-                !this_pr_can_use_special_jll_exceptions),
-            (guideline_name_length,                             # 5
-                !this_pr_can_use_special_jll_exceptions),
-            (guideline_julia_name_check, true),                 # 6
-            # (guideline_standard_initial_version_number,       # 7 (deactivated)
-            #     !this_pr_can_use_special_jll_exceptions),
-            (guideline_repo_url_requirement, true),             # 8
-            (guideline_compat_for_julia, true),                 # 9
-            (guideline_compat_for_all_deps, true),              # 10
-            (guideline_allowed_jll_nonrecursive_dependencies,   # 11
-                this_is_jll_package),
-            (guideline_name_ascii, true),                       # 12
-            (:update_status, true),
-            (guideline_version_can_be_pkg_added, true),         # 13
-            (guideline_code_can_be_downloaded, true),           # 14
-            # `guideline_version_has_osi_license` must be run
-            # after `guideline_code_can_be_downloaded` so
-            # that it can use the downloaded code!
```

Comment:
I think it's unfortunate to lose these comments.

Reply:
Good call. I've restored those comments.